src/index.html

```diff
@@ -1,12 +1,377 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="moyo_logo" >
+      </a>
+      <nav class="header__nav">
+        <ul class="nav">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
don't forget about empty line between siblings
```suggestion
          </li>
          
          <li class="nav__item">
```